homeassistant/components/media_player/lg_netcast.py

```diff
@@ -104,7 +104,8 @@ def update(self):
                 channel_list = client.query_data('channel_list')
                 if channel_list:
                     channel_names = [str(c.find('chname').text) for
```

Comment:
This is better done in a normal for loop now as now you're executing `c.find` twice.